Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -214,6 +248,26 @@ public class GraphLinter: GraphLinting {
         return issues
     }
 
+    private func lintAppClipsBundleIdentifier(graph: Graph) -> [LintingIssue] {
```

Comment:
minor: To avoid traversing the graph multiple times to extract app / appClip pair, could the lookup be done once and then the app / app clips be passed onto the individual linting methods? 

e.g.

```swift
func lintAppClips(graph: Graph) -> [LintingIssue] {
   // perform graph lookup
   let apps = // ...

   // ... 
   issues.append(contentsOf:
       lintAppClipsBundleIds(apps: apps)
   )
   // ...
}
```

Reply:
sure, will address this